Development.md

```diff
@@ -0,0 +1,163 @@
+# Development
+
+## Running from a Git checkout
+
+If you want to run on the bleeding edge, or [help contribute to Meteor](Contributing.md), you
+can run Meteor directly from a Git checkout using these steps:
+
+0. **Clone from GitHub**
+
+    ```sh
+    $ git clone --recursive https://github.com/meteor/meteor.git
+    $ cd meteor
+    ```
+
+    > ##### Important note about Git submodules!
+    >
+    > This repository uses Git submodules.  If you clone without the `--recursive` flag,
+    > re-fetch with `git pull` or experience "`Depending on unknown package`" errors,
+    > run the following in the repository root to sync things up again:
+    >
+    >     $ git submodule update --init --recursive
+
+0. **_(Optional)_ Compile dependencies**
```

Comment:
I do this quite often, but you're right that it should be in a separate section.